### ISSUE TITLE
Blue text in many links and buttons does not meet WCAG color contrast standards

### ISSUE THREAD

**Issue by hs4man21** [bluesky-social/social-app#629]
**Describe the bug**
The blue font color used throughout Bluesky's apps for much of the interactive text (links and buttons) has insufficient color contrast in light mode (contrast of 3.62:1) and dark mode. The text fails WCAG AA in light mode. The contrast is better in dark mode (5.79:1), which passes WCAG AA compliance but still fails [WCAG AAA compliance (7+:1 contrast for "normal text")](https://www.w3.org/WAI/WCAG21/Understanding/contrast-enhanced.html).

**To Reproduce**

1. I used inspect element in Google Chrome on the footer text in the Bluesky web app (see below screenshot) to confirm the contrast was below compliance levels.
2. [The free WebAIM color contrast checker tool](https://webaim.org/resources/contrastchecker/) can be used to test alternative colors for potential use.

**Expected behavior**
Text, graphical objects and UI components should pass the WCAG AAA threshold for color contrast, as reflected by [the free WebAIM color contrast checker tool](https://webaim.org/resources/contrastchecker/).

**Screenshots**
<img width="351" alt="Screenshot 2023-05-15 at 5 56 19 PM" src="https://github.com/bluesky-social/social-app/assets/38192823/8b258c4c-e253-4911-8715-7ccb5e98dab4">

**Details**

 - Platform: All devices and browsers
 - Platform version: N/A
 - App version: 1.28 (and previous versions)

**Comment by mozzius** [comment]
Fixed by #9030 I believe!
